_includes/feature-image.html

```diff
@@ -1,11 +1,16 @@
-{% assign include_image = image | default: page.image %}
-{% assign include_image_alt = image | default: page.image_alt | default: "" %}
-{% assign include_figcaption = image_figcaption | default: page.image_figcaption %}
-
-{% if page.hero %}
-<figure class="post-feature_image" role="img" title="{{ include_image_alt }}" style="background-image: url('{{ site.baseurl }}{{ include_image }}')">
-  {% if include_figcaption %}
-    <figcaption>{{- include_figcaption -}}</figcaption>
-  {% endif %}
-</figure>
+{% if image %}
+  {% comment %}
+    I'm wondering if there's a better way to do this, like setting up
+    a shortcode.
```

Comment:
That's fair, this was just me writing stream-of-consciousness while coding.

This one in particular feels ok to me to leave as a comment, or I could delete it. It's not _really_ necessary, but thought it could be nice to validate a future reader who comes by with the same thought.

Reply:
sure, sgtm